sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobSasImplUtil.java

```diff
@@ -112,6 +114,21 @@ public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String container
      */
     public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String containerName, String blobName,
         String snapshotId, String versionId) {
+        this(sasValues, containerName, blobName, snapshotId, versionId, null);
+    }
```

Comment:
I think I can just change the signature directly since this is in implementation

Reply:
+1

Reply:
done